### ISSUE TITLE
Icon size appears very small

### ISSUE THREAD

**Issue by robertguetzkow** [doctormo/GimpPs#16]
The current settings use a very small icon size for the tools. The default value in gtkrc is `GimpToolPalette::tool-icon-size  = menu` and produces icons which are significantly smaller than the saved icons. Setting `GimpToolPalette::tool-icon-size  = small-toolbar` seems closer to the look of Adobe Photoshop.

**Comment by doctormo** [comment]
This does seem to depend on the resolution of the display. What resolutions are your monitors and do you know their true dpi resolution?

**Comment by robertguetzkow** [comment]
I thought the icons would be scaled relative to the display resolution. My resolution is 1920 x 1080 px and 166 ppi (according to http://dpi.lv/).

**Comment by doctormo** [comment]
No, icons are scaled to a fixed metric. HiDPI is a setting for instance that doubles the metric for all graphics. So for example if the icons are 16px by 16px, on your screen they will look 1/10th of an inch or 2.5mm and that's tiny. But with a HiDPI setting enabled, they'd be 5mm instead.

There are a couple of different settings that change the metric for the icons, for example setting the /type/ of image from menu to button or to image. But that has to be done carefully as setting it too big will leave other people unable to use gimp at all.

**Comment by burbleflurp** [comment]
Thanks so much for this - have just run it under Ubuntu and certainly looks fine.....
except I'm a noob (looking to learn/use gimp) and was struggling with gimp's icons.
Can you recommend a way to increase the size of the icons by any chance?
I can identify that my screen resolution is 1920x1080 at 166ppi using the dpi.lv link.

**Comment by doctormo** [comment]
Honestly, I'd start by trying to make sure your Gnome settings are set right. Do other programs look small too? Or is it just gimp with this theme?

**Comment by burbleflurp** [comment]
Thanks for answering so quickly. Just to correct an misinterpretation - I'm not sure that anything is wrong with the theme at all (ie in its installation or how it displays), its just that my eyes (and the rest of me) is getting older and would be grateful for some larger icons. Searching the internet has shown that there are a number of GIMP themes developed to specifically provide larger icons. They seem to do it just by increasing each icon size from 16x16 to 32x32. If I'm right, I'll try changing the icon size in the tools folder and see what happens. As I haven't played with GIMP themes before. I don't know whether anything else needs to be altered to adjust things. Oh, and thanks for creating this package.

**Comment by burbleflurp** [comment]
Resizing 16x16 tool icons to 24x24, and altering Psicons gtkrc to read
GimpToolPalette::tool-icon-size   = button
solved all my problems. Thanks for the wonderful work you have done.

**Comment by kadamwhite** [comment]
Some sort of out-of-the-box support for higher-resolution or HiDPI screens would be awesome; or at least a callout in the readme or such explaining how to control the icon sizing. This is an absolutely amazing project, thanks so much!

**Comment by hubyhuby** [comment]
I have the same issue,
With the original gimp I have the tinyyyy icones. I have to bring my nose on the screen to read it.
With your wonderfull mode gimp s gets usable, also it keeps the tiny icones.
I have a very standard screen of 1920X 1080. I believe that it is the bare minimum for graphists today.
Tried to copy past the previous lines in my "gtkrc" file with no luck.

Would be nice to have it as a standard in this mode.

Thanks for making gimp user friendely. Wished it would be the gimp default.
